### PR TITLE
Remove unnecessary quotes from front-end pages and email templates; 

### PR DESCRIPTION
**Pull request by renzms**
see websharks/comment-mail#201

### CODE REVIEW

**Comment by raamdev** [comment]
@renzms I left you feedback here: https://github.com/websharks/comment-mail/issues/201#issuecomment-174598238

**Comment by renzms** [comment]
@raamdev Updated with <em> tags for Lite version

File: comment-mail/includes/templates/type-s/email/comment-notification/snippet/message-heading.php [hunk 1]
```diff
@@ -4,24 +4,24 @@
...
 			[if subscribed_to_own_comment]
-				New Replies to Your Comment on "[sub_post_title_clip]"
+				New Replies to Your Comment on <em>[sub_post_title_clip]</em>
 			[else]
-				New Replies to <a href="[sub_comment_url]">a Comment</a> on "[sub_post_title_clip]"
+				New Replies to <a href="[sub_comment_url]">a Comment</a> on <em>[sub_post_title_clip]</em>
 			[endif]
 		[else]
-			New Comments on "<a href="[sub_post_comments_url]">[sub_post_title_clip]</a>"
+			New Comments on <a href="[sub_post_comments_url]"><em>[sub_post_title_clip]</em></a>
```
Comment (raamdev) [inline comment-mail/includes/templates/type-s/email/comment-notification/snippet/message-heading.php]: @renzms Can you please move the `<em></em>` markup _outside_ the anchor element on this line? You already did that everywhere else; this was the only line you did differently.

**Comment by raamdev** [comment]
@renzms I left some feedback above. ↑ Please also update the Pro PR.

**Comment by renzms** [comment]
@raamdev 

Updated, ready for check. Thanks!

**Pull request closed without merging**